packages/serverpod/test/database/database_query_test.dart

```diff
@@ -0,0 +1,399 @@
+import 'package:serverpod/database.dart';
+import 'package:serverpod/src/database/database_query.dart';
+import 'package:serverpod/src/database/table_relation.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Given SelectQueryBuilder', () {
+    test('when default initialized then build outputs a valid SQL query.', () {
+      var query = SelectQueryBuilder(table: 'citizen').build();
+
+      expect(query, 'SELECT * FROM citizen');
```

Comment:
perhaps table names should be in quotes too, just to be safe? (Not sure what the current situation is.)

Reply:
Implemented! 👍